awx/ui_next/src/screens/Inventory/InventoryGroups/InventoryGroupDetail/InventoryGroupDetail.jsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from 'react';
+import { t } from '@lingui/macro';
+
+import { CardBody, Button } from '@patternfly/react-core';
+import { withI18n } from '@lingui/react';
+import { withRouter, Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { VariablesInput as CodeMirrorInput } from '@components/CodeMirrorInput';
+import ErrorDetail from '@components/ErrorDetail';
+import AlertModal from '@components/AlertModal';
+import { formatDateString } from '@util/dates';
+
+import { GroupsAPI } from '@api';
+import { DetailList, Detail } from '@components/DetailList';
+
+const VariablesInput = styled(CodeMirrorInput)`
+  .pf-c-form__label {
+    font-weight: 600;
+    font-size: 16px;
+  }
+  margin: 20px 0;
+`;
+const ActionButtonWrapper = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 20px;
+  & > :not(:first-child) {
+    margin-left: 20px;
+  }
+`;
+function InventoryGroupDetail({ i18n, history, match, inventoryGroup }) {
+  const {
+    summary_fields: { created_by, modified_by },
+    created,
+    modified,
+  } = inventoryGroup;
+  const [error, setError] = useState(false);
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+
+  const handleDelete = async () => {
+    setIsDeleteModalOpen(false);
+    try {
+      await GroupsAPI.destroy(inventoryGroup.id);
+      history.push(`/inventories/inventory/${match.params.id}/groups`);
+    } catch (err) {
+      setError(err);
+    }
+  };
+
+  let createdBy = '';
+  if (created) {
+    if (created_by && created_by.username) {
+      createdBy = (
+        <span>
+          {i18n._(t`${formatDateString(inventoryGroup.created)} by`)}{' '}
+          <Link to={`/users/${created_by.id}`}>{created_by.username}</Link>
+        </span>
+      );
+    } else {
+      createdBy = i18n._(t`${formatDateString(inventoryGroup.created)} by`);
```

Comment:
What is the value of `createdBy` in the else block? `10/10/2009, 1:17:17 PM by`? I think we should remove the string " by". 

This also applies for `modifiedBy` below.